app/policies/admin/admin/github_user_policy.rb

```diff
@@ -1,4 +1,4 @@
-class Admin::GitHubUserPolicy < ApplicationPolicy
+class Admin::Admin::GitHubUserPolicy < Admin::ApplicationPolicy
```

Comment:
Before you cringe at this, I have to say that I thought about this and this is the "automatic" way of referencing this policy in the same way all other admin policies are loaded. I don't like it either so if you have an idea that would make this better, I'm open to hearing it.

Reply:
Could we keep this policy in `policies/admin` and put a manual override to avoid this?

Reply:
Ok, I made a better version of this. I'm happy with it.